tests/unit/client/test_artifact_client.py

```diff
@@ -26,6 +29,23 @@ def test_get_data(project_client):
     assert artifact.data == data
 
 
+def test_internal_get_data_multiple_backend_error():
```

Comment:
this and `test_get_data_multiple_backend_error` on line 74 seem to be identical - am I missing something?

Reply:
Hey! This line `test_get_data_multiple_backend_error` on line 74 is testing `Artifact.get_data()` and the line 84 `test_internal_get_data_multiple_backend_error` is testing the private `Artifact._get_data()`

Reply:
gotcha, thanks for being thorough!